configuration/verify-service-provider.yml

```diff
@@ -25,6 +25,7 @@ clockSkew: ${CLOCK_SKEW:-PT5s}
 
 # Entity ID that uniquely identifies your service
 serviceEntityId: ${SERVICE_ENTITY_ID:-http://verify-service-provider}
+assertionConsumerServiceUri: ${ASSERTION_CONSUMER_SERVICE_URI:-http://localhost:3200/verify/response}
```

Comment:
Can we put a comment above this one explaining what it is (like the others)? Something like:

"The URI that Verify should send SAML responses to"

Reply:
Yes. We also want to discuss about the sensible default value.